server/services/reportStatusChecker.test.js

```diff
@@ -2,7 +2,16 @@ const { SectionStatus, check } = require('./reportStatusChecker')
 
 describe('statusCheck', () => {
   const validReport = {
-    details: {
+    incidentDetails: {
+      witnesses: [{ name: 'BOB BARRY' }, { name: 'JAMES JOHN' }],
+      locationId: -25,
+      involvedStaff: [
+        { name: 'Itag User', email: 'itag_user@digital.justice.gov.uk', staffId: 1, username: 'ITAG_USER' },
```

Comment:
out of interet why is `staffId` coming through here?  Are you using it at all - note that auth users won't have one...

Reply:
As discussed - I'll add a ticket to make this optional in the short term with the hope to move to a better required, immutable identifier in the future when one become available.